docs/element-definitions.md

```diff
@@ -1118,6 +1118,53 @@ Data structure definition using Data Structure elements.
 - `"messageBody"` - Asset is an example of message-body
 - `"messageBodySchema"` - Asset is a schema for message-body
 
+### Server ([Object][])
+
+Subtype of [Object][] representing a server.
+
+#### Template
+
+- `element` - `"server"`
+- `attributes`
+    - `href` ([Templated Href][]) - URI Template for this server.
+    - `hrefVariables` ([Href Variables][]) - URI Template variables.
+
+#### Example
+
+```json
+{
+  "element": "server",
+  "meta": {
+    "description": {
+      "element": "string",
+      "content": "Description of the server."
```

Comment:
```suggestion
      "content": "Production"
```

I'd suggest using an actual example description to make this example a bit more real-world.